### PR TITLE
V1.3

### PR DESCRIPTION
**Pull request by Si-Shi**
- Updated master network
- Updated 2050 project lists
- Added 2030 and 2040 CAMPO project list
- Change gitignore file so that 2030 and 2040 folder shows up

### CODE REVIEW

File: scenarios/.gitignore [hunk 1]
```diff
@@ -1,4 +1,6 @@
 *
 !.gitignore
 !base_2020/
-!mtp_2050/
\ No newline at end of file
+!2030/
+!2040/
+!2050_Amendment/
```
Comment (dkyleward) [inline scenarios/.gitignore:6]: Why create a 2050_Amendment? It's still the 2050 scenario right? The amendment is handled by git, so I'd suggest just changing the mtp_2050 folder. Unless you guys specifically want to people to be able to run the old 2050 and the amended. In my mind, this just seems confusing for consultants/public.

File: src/utils/deletefiles.rsc [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        
+        for i = 1 to files.length do
+            file = files[i][1]
+            filepath = dir + "/" + file
+            DeleteFile(filepath)
+        end
+        if GetDirectoryInfo(dir, "All") <> null then RemoveDirectory(dir)
+    end
+
+endmacro
```
Comment (dkyleward) [inline src/utils/deletefiles.rsc:66]: Nice!

**Review by dkyleward — CHANGES_REQUESTED** [review]
Looks good. I only have one comment that needs addressing. Please take a look.

**Comment by Flatiron87** [comment]
It’s possible we would want to run the MTP 2050 adopted or at least know which projects were included in the adopted 2050.  I don’t care how that is tracked, but the current project list needs to be maintained (perhaps identifying it as a release?), as well as the MTP 2050 amendment project list..
Mike

Mike Bruff
Transportation Modeling Engineer
Capital Area MPO
One Fenton Main Street, Suite 201
Cary, NC 27511
984-542-3613 (office)

campo-nc.us<https://www.campo-nc.us/>
***@***.******@***.***>

File: scenarios/.gitignore (hunk 1 above)
Comment (Si-Shi) [reply scenarios/.gitignore:6]: OK!

**Comment by Si-Shi** [comment]
Thanks for your feedback, Mike. We will keep both 2050 and 2050 Amendment scenarios.

**Pull request closed without merging**